src/main/java/com/smartnews/jpa_entity_generator/metadata/TableMetadataFetcher.java

```diff
@@ -31,7 +31,7 @@ private DatabaseMetaData getMetadata(JDBCSettings settings) throws SQLException
         DatabaseMetaData databaseMeta = getMetadata(jdbcSettings);
         try {
             List<String> tableNames = new ArrayList<>();
-            try (ResultSet rs = databaseMeta.getTables(null, null, "%", TABLE_TYPES)) {
+            try (ResultSet rs = databaseMeta.getTables(null,  jdbcSettings.getSchemaName(), "%", TABLE_TYPES)) {
```

Comment:
```
ResultSet getTables​(String catalog, String schemaPattern, String tableNamePattern, String[] types) throws SQLException
```
https://docs.oracle.com/en/java/javase/11/docs/api/java.sql/java/sql/DatabaseMetaData.html#getTables(java.lang.String,java.lang.String,java.lang.String,java.lang.String%5B%5D)

According to the javadoc, it's not a concrete schema name. `JDBCSettings#getSchemaPattern` may be more suitable name for it.